secure_fw/partitions/initial_attestation/attest_token_encode.c

```diff
@@ -241,6 +241,9 @@ attest_token_encode_start(struct attest_token_encode_ctx *me,
     me->opt_flags  = opt_flags;
     me->key_select = key_select;
 
+    if (opt_flags & TOKEN_OPT_SIGN_MESSAGE) {
```

Comment:
Should I make sure this is only added for nRF54L15, or should we use sign_message for all chips?